src/app/features/charts/components/median-barchart/median-barchart.component.ts

```diff
@@ -0,0 +1,260 @@
+        .append('g')
+        .attr('transform', `translate(${margin.left}, ${margin.top})`);
+
+      // SCALES
+      const xScale = d3
+        .scaleBand()
+        .domain(this._data.map((d) => d.tissue))
+        .range([0, innerWidth])
+        .padding(0.2);
+
+      const yScale = d3
+        .scaleLinear()
+        .domain([0, this.maxValueY])
+        .nice()
+        .range([innerHeight, 0]);
+
+      // BARS
+      this.chart
+        .selectAll('.medianbars')
+        .data(this._data)
+        .enter()
+        .append('rect')
+        .attr('class', 'medianbars')
+        .attr('x', (d) => xScale(d.tissue) as number)
+        .attr('y', (d) => yScale(d.medianlogcpm || 0))
+        .attr('width', xScale.bandwidth())
+        .attr('height', (d) => innerHeight - yScale(d.medianlogcpm || 0))
+        .attr('fill', barColor);
+
+      // SCORE LABELS
+      this.chart
+        .selectAll('.bar-labels')
+        .data(this._data)
+        .enter()
+        .append('text')
+        .attr('class', 'bar-labels')
+        .attr('x', (d) => this.getBarCenterX(d.tissue, xScale))
+        .attr('y', (d) => yScale(d.medianlogcpm || 0) - 5)
+        .text((d) => this.helperService.roundNumber(d.medianlogcpm || 0, 2));
+
+      // X-AXIS
+      const xAxis = d3.axisBottom(xScale);
+      this.chart
+        .append('g')
+        .attr('class', 'x-axis')
+        .attr('transform', `translate(0, ${innerHeight})`)
+        .call(xAxis.tickSizeOuter(0))
+        .selectAll('.tick')
+        .on('mouseenter', (_, tissue) => {
+          const tooltipText = this.helperService.getGCTColumnTooltipText(
+            tissue as string
+          );
+          this.showTooltip(
+            tooltipText,
+            this.getBarCenterX(tissue as string, xScale) + margin.left,
+            height - margin.top
+          );
+        })
+        .on('mouseleave', () => {
+          this.hideTooltip();
+        });
+
+      // Y-AXIS
+      const yAxis = d3.axisLeft(yScale);
+      this.chart.append('g').attr('class', 'y-axis').call(yAxis);
+
+      // X-AXIS LABEL
+      this.chart
+        .append('text')
+        .attr('class', 'x-axis-label')
+        .attr('x', innerWidth / 2)
+        .attr('y', innerHeight + margin.bottom)
+        .attr('text-anchor', 'middle')
+        .text(this.xAxisLabel);
+
+      // Y-AXIS LABEL
+      this.chart
+        .append('text')
+        .attr('class', 'y-axis-label')
+        .attr('x', -innerHeight / 2)
+        .attr('y', -margin.left)
+        .attr('dy', '1em')
+        .attr('text-anchor', 'middle')
+        .attr('transform', 'rotate(-90)')
+        .text(this.yAxisLabel);
+
+      // THRESHOLD LINE
+      this.chart
+        .append('line')
+        .attr('class', 'meaningful-expression-threshold-line')
+        .attr('x1', 0)
+        .attr('x2', innerWidth)
+        .attr('y1', yScale(this.MEANINGFUL_EXPRESSION_THRESHOLD))
+        .attr('y2', yScale(this.MEANINGFUL_EXPRESSION_THRESHOLD))
```

Comment:
Based on [the chart description](https://github.com/Sage-Bionetworks/Agora/blob/caf6888e874ca9a4d5011d9813d446019a921a62/src/app/features/genes/components/gene-evidence-rna/gene-evidence-rna.component.html#L33), I expected that the threshold line would always be shown at log2(5), which is ~2.3. However [on develop](https://agora-develop.adknowledgeportal.org/genes/ENSG00000183856/evidence/rna), when the max value of the chart is close to the threshold value, the threshold value is drawn at slightly less than 2. 

I used `yScale` to calculate the position for the threshold, so it is drawn at log2(5) in this case. However, I might've misunderstood how the threshold should be calculated, so happy to adjust as needed!

develop | feature 
:---:|:---:
<img width="1770" alt="main_ENSG00000183856_closeThreshold" src="https://github.com/Sage-Bionetworks/Agora/assets/26949006/aeab3c23-10f1-4e87-b0b6-0b9aa03be336">  | <img width="1774" alt="feature_ENSG00000183856_closeThrehsold" src="https://github.com/Sage-Bionetworks/Agora/assets/26949006/a900a72b-fa56-439e-996e-da31d6a76971">

Reply:
Another great question for @JessterB.  I haven't looked at this chart so I'm not 100% sure.

Reply:
The threshold should be drawn at log2(5), which is what I see running this locally.